src/components/betting-chips/BettingControls.module.scss

```diff
@@ -1,25 +1,191 @@
+    height: 100%;
+    border: 4px solid rgb(70, 16, 18);
+    border-bottom: none;
+    color: var(--betting-font-color);
+
+    > p {
+      font-size: 2rem;
+      color: rgba(250, 250, 250, 0.6);
+      margin: 0;
+      margin-bottom: 0.1rem;
+    }
+  }
 
-    .player, .tie, .banker {
-        height: 10rem;
-        background: rgb(164, 54, 54);
-        border: 2px solid orange;
+  .fieldInfo {
+    margin-top: 1rem;
+    .name,
+    .stake {
+      margin: 0;
     }
 
-    .player {
-        flex-basis: 40%;
-        border-width: 2px 0 2px 2px;
+    .name {
+      font-size: 1.5rem;
     }
 
-    .tie {
-        flex-basis: 20%;
+    .stake {
+      color: rgba(250, 250, 250, 0.5);
     }
+  }
+
+  .player {
+    flex-basis: 40%;
+    border-width: 4px 0 0 0;
+  }
+
+  .tie {
+    flex-basis: 20%;
+  }
+
+  .banker {
+    flex-basis: 40%;
+    border-width: 4px 0 0 0;
+  }
+}
+
+.controls {
+  padding: 0 1rem 1rem 1rem;
+  display: flex;
+  align-items: baseline;
+  justify-content: center;
+  gap: 0.5rem;
+  grid-row: 3 / 3;
+  grid-column: 1 / span 3;
+  z-index: 300;
+}
+
+.small {
+  height: 3rem;
+  width: 3rem;
+  font-size: 1.1rem;
+}
+
+.big {
+  width: 5rem;
+  height: 5rem;
+  font-size: 1.5rem;
+}
+
+.btn {
+  cursor: pointer;
+  display: flex;
+  align-items: center;
+  justify-content: center;
+  background: radial-gradient(
+      ellipse farthest-corner at right bottom,
+      #fedb37 0%,
+      #fdb931 8%,
+      #9f7928 30%,
+      #8a6e2f 40%,
+      transparent 80%
+    ),
+    radial-gradient(
+      ellipse farthest-corner at left top,
+      #ffffff 0%,
+      #ffffac 8%,
+      #d1b464 25%,
+      #5d4a1f 62.5%,
+      #5d4a1f 100%
+    );
+  border-radius: 100%;
+  border: none;
+  outline: none;
+  padding: 0;
+  box-shadow: 1px 1px 5px rgba(0, 0, 0, 1);
 
+  &:hover {
+    filter: brightness(95%);
+  }
+
+  &:disabled {
+    filter: brightness(60%);
+    cursor: default;
+  }
+
+  .inside {
+    display: flex;
+    align-items: center;
+    justify-content: center;
+    background-color: rgba(0, 0, 0, 0.8);
+    color: rgba(254, 255, 191);
+    text-shadow: 0px 0px 12px rgb(254 255 191 / 40%);
+    font-size: 1.5rem;
+    width: 85%;
+    height: 85%;
+    border-radius: 100%;
+  }
+
+  .innermost {
+    display: flex;
+    align-items: center;
+    justify-content: center;
+    background: linear-gradient(
+      to right bottom,
+      rgb(230, 217, 135) 0%,
+      rgb(207, 155, 42) 60%,
+      rgb(121, 98, 40) 100%
+    );
+    color: rgba(254, 255, 191);
+    text-shadow: 0px 0px 12px rgb(0 0 0 / 90%);
+    font-size: 1.5rem;
+    width: 97%;
+    height: 97%;
+    box-shadow: inset 0px 0px 4px 2px rgb(0 0 0 / 70%);
+    border-radius: 100%;
+  }
+}
+
+.spin {
+  animation-name: spin;
+  animation-duration: 0.7s;
+  animation-iteration-count: 1;
+}
+
+@keyframes spin {
+  from {
+    transform: rotateY(0deg);
+  }
+  to {
+    transform: rotateY(360deg);
+  }
+}
+
+$narrow-screen: 650px;
```

Comment:
These styles are also repeated, you can move them to [mixins](https://sass-lang.com/documentation/at-rules/mixin)